### DISCUSSION TITLE
Marker over a black background

### DISCUSSION THREAD

**Discussion by MoKoChino** [flameshot-org/flameshot#3192] [Q&A] [answered]
Hi!
Has someone noticed that when you use the marker over a black background it's barely seen? Is there any configuration I'm missing to make this more visible?
If not possible yet, maybe it can be added as a New Feature in order to provide the capability to increase/decrease the transparency for the marker.
Thanks!

**Comment by mmahmoudian** [comment, marked as answer]
Adding colors on top of a back background is in general a hard thing to see. I suggest you use invert color for those parts to attract attention

**Reply by MoKoChino** [reply]
That's a great workaround! I'll use that in the future. I've also noticed that if I use the marker several times on top of each, it gets better...

Flameshot:
![flameshot](https://github.com/flameshot-org/flameshot/assets/31129110/37a197ae-3786-42e1-a63e-346af545b0a3)


But also, other tools like 'Lightshot' can make it with just one draw and works quite well, so it shouldn't be hard to implement.
Lightshot:
![lightshot](https://github.com/flameshot-org/flameshot/assets/31129110/2c4e4bc3-239e-4aeb-a7e3-67244654e822)

**Comment by mmahmoudian** [comment]
I think this kinda points to the direction of #2653

**Reply by FuryBart** [reply]
And even more this one: https://github.com/flameshot-org/flameshot/issues/3007
